tests/unit/test_computations.py

```diff
@@ -162,7 +162,7 @@ def test_moving_median(window_size: int, expected_output: np.ndarray) -> None:
     """Testing proper function of moving median function."""
     dummy_date = datetime(2024, 5, 2)
     dummy_datetime_list = [dummy_date + timedelta(seconds=i) for i in range(3)]
-    dummy_datetime_pl = pl.Series("time", dummy_datetime_list)
+    dummy_datetime_pl = pl.Series(dummy_datetime_list)
```

Comment:
would anything bad happen if it was assigned the name "time" here? just redundant since the moving_median enforces it

Reply:
It'd just be ignored